packages/block-library/src/query/index.php

```diff
@@ -51,8 +55,8 @@ function render_block_core_query( $attributes, $content, $block ) {
 			// Add the necessary directives.
 			$p->set_attribute( 'data-wp-interactive', 'core/query' );
 			$p->set_attribute( 'data-wp-router-region', 'query-' . $attributes['queryId'] );
-			$p->set_attribute( 'data-wp-init', 'callbacks.setQueryRef' );
 			$p->set_attribute( 'data-wp-context', '{}' );
+			$p->set_attribute( 'data-wp-key', $attributes['queryId'] );
```

Comment:
I'm also wondering about this change that gets applied when the experiment is both enabled and disabled. Was `setQueryRef` in use before? I can't find it in the source code.

Reply:
From what I understood from @DAreRodz , it was part of an old implementation, and it seems it wasn't removed. I couldn't find it either.

It seems the `setQueryRef` callback:

* Was added [here](https://github.com/WordPress/gutenberg/commit/19b2bf9cc51074e3bcc9448689618912c391ab5d).
* The JS logic was removed [here](https://github.com/WordPress/gutenberg/pull/58496/files), but we didn't remove the directive.

Reply:
There is an issue planning to unset the queryId if the query inherits its options.
https://github.com/WordPress/gutenberg/issues/60374